lib/miq_automation_engine/service_models/miq_ae_service_authentication_rhsm.rb

```diff
@@ -0,0 +1,5 @@
+module MiqAeMethodService
+  class MiqAeServiceAuthenticationRhsm < MiqAeServiceAuthentication
```

Comment:
None of them will get to automate because all of the other are defined as identity authentication, I never call them in the automate, and if ill need it ill add move service models.